reports/cveinfo/Dockerfile

```diff
@@ -22,11 +22,9 @@ ENV COSMOS_KEY="" \
     COSMOS_DB_NAME="" \
     COSMOS_DB_CONTAINER="" \
     COSMOS_DB_URI="" \
-    SECRET_PATH="" \
-    VAULT_NAME="" \
     MAX_BATCH_SIZE=1000
 
-RUN chmod +x /app/set_env.sh
+# RUN chmod +x /app/set_env.sh
```

Comment:
Why are we leaving #comments in - just remove?

Reply:
need it to serve as a ref